### DISCUSSION TITLE
Updating RDKit

### DISCUSSION THREAD

**Discussion by BernalFA** [rdkit/rdkit#4147] [General]
Hello everyone,

I have been working with RDKit on both Windows and Ubuntu systems and I would like to update to the most recent version to make use of the new features included. However, I could not find any clear information about it. Should I remove the old version and install the new one? And what about the environment? This would require to create a new environment (as typically done according the installation section of the RDKit documentation)?   

I would really appreciate if you can help me.

Thanks

**Comment by greglandrum** [comment]
@BernalFA : sorry for the slow reply here
If you're using conda with the rdkit (which I highly recommend) you just need to use conda's update command:
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html#updating-packages

**Comment by vfscalfani** [comment]
Hi @BernalFA. Another approach would be to create a new environment (use a different name) and install the latest rdkit as per the docs. That's usually what I do when major releases of RDKit come out, this way I can keep the older known working environment, and then remove it after verifying my new environment is all setup. To remove environments, see here: https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html
